scanpointgenerator/excluders/roiexcluder.py

```diff
@@ -36,9 +36,12 @@ def create_mask(self, x_points, y_points):
 
         mask = np.zeros_like(x_points, dtype=bool)
         for roi in self.rois:
+            x = x_points.copy()
```

Comment:
I'd leave a comment reminding people that the regions perform in-place operations on the point arrays.